sklearn/cluster/_k_means_elkan.pyx

```diff
@@ -18,7 +18,7 @@ from ..metrics import euclidean_distances
 from ._k_means import _centers_dense
 
 
-cdef floating euclidian_dist(floating* a, floating* b, int n_features) nogil:
+cdef floating euclidean_dist(floating* a, floating* b, int n_features) nogil:
```

Comment:
It looks private. The outputs of finding all files containing `euclidian_dist` are as follows:
```
$ grep -r "euclidian_dist(" .
./sklearn/cluster/_k_means_elkan.pyx:cdef floating euclidian_dist(floating* a, floating* b, int n_features) nogil:
./sklearn/cluster/_k_means_elkan.pyx:        d_c = euclidian_dist(x, centers, n_features)
./sklearn/cluster/_k_means_elkan.pyx:                dist = euclidian_dist(x, c, n_features)
./sklearn/cluster/_k_means_elkan.pyx:                        upper_bound = euclidian_dist(x_p, centers_p + label * n_features, n_features)
./sklearn/cluster/_k_means_elkan.pyx:                        distance = euclidian_dist(x_p, centers_p + center_index * n_features, n_features)
```